lib/ClangImporter/ImporterImpl.h

```diff
@@ -275,6 +275,7 @@ class LLVM_LIBRARY_VISIBILITY ClangImporter::Implementation
 
   bool IsReadingBridgingPCH;
   llvm::SmallVector<clang::serialization::SubmoduleID, 2> PCHImportedSubmodules;
+  llvm::SmallVector<const clang::Module*, 2> ImportedModules;
```

Comment:
Can you name this DeferredHeaderImports or something? ImportedModules makes it sound like something that stays valid and makes sense to ask for for the lifetime of the ClangImporter.